### DISCUSSION TITLE
any examples of using uwebsocket as...

### DISCUSSION THREAD

**Discussion by hiqsociety** [uNetworking/uWebSockets#957] [Q&A] [answered]
1. any examples of using uwebsocket as reverse proxy?
2. extracting header of http?
3. co routine processing?
thx in advance

**Comment by ghost** [comment, marked as answer]
A reverse proxy is much more than what this project gives you out of the box

**Comment by hiqsociety** [comment]
Any hints on which modules you will recommend to accomplish the task needed? as part of the recommended extension to the programs mentioned.

**Comment by ghost** [comment]
No that's out of scope for this project right now. This project only implements a server
